doc-deploy-dev/action.yml

```
@@ -147,6 +159,8 @@ runs:
     - name: "Ensure that the desired branch exists"
       env:
         BRANCH: ${{ inputs.branch }}
+        BOT_USER: ${{ inputs.bot-user }}
+        BOT_EMAIL: ${{ inputs.bot-email }}
       shell: bash
```

```suggestion
      shell: bash
```
Same applies here but you would have to search for the env.BRANCH and change it... GH-s review view does not let me modify it myself